File: 1979-find-greatest-common-divisor-of-array/1979-find-greatest-common-divisor-of-array.java
class Solution {
    public int findGCD(int[] nums) {
        Arrays.sort(nums);
        int o = nums[0];
        int e = nums[nums.length-1];


        while (o != 0) {
            int temp = o;
            o = e % o;
            e = temp; 
        }
        return e;

        
    }
}

File: 1979-find-greatest-common-divisor-of-array/1979-find-greatest-common-divisor-of-array.js
var findGCD = function (nums) {
    nums.sort((a, b) => a - b);

    let o = nums[0];
    let e = nums[nums.length - 1];

    while (o !== 0) {
        let temp = o;
        o = e % o;
        e = temp;
    }

    return e;
};